package.json

```diff
@@ -42,9 +43,10 @@
     "@volar-plugins/vetur": "latest",
     "@vue/eslint-config-standard": "^4.0.0",
     "@vue/eslint-config-typescript": "^9.1.0",
-    "@vue/test-utils": "^1.3.5",
+    "@vue/test-utils": "^2.4.0-alpha.2",
     "axios-mock-adapter": "^1.21.5",
     "eslint": "~8.50.0",
+    "eslint-plugin-import": "^2.28.1",
```

Comment:
What's this for?

Reply:
Something I added for resolving an issue with imports but I think this is not useful and will remove it.